### DISCUSSION TITLE
Support Github enterprise for custom templates

### DISCUSSION THREAD

**Discussion by shahbagdadi** [projectdiscovery/nuclei#4135] [Ideas]
Users can create custom templates on a personal public / private GitHub / AWS Bucket that they wish to run / update while using nuclei from any environment without manually downloading the GitHub repository everywhere.

**Can you please also add support for Github Enterprise ? You may need to allow the user to set GITHUB_SERVER_URL url**

**Comment by ehsandeep** [comment]
should be supported already, see https://github.com/projectdiscovery/nuclei/blob/main/v2/cmd/nuclei/issue-tracker-config.yaml#L9

**Comment by shahbagdadi** [comment]
Is that for issue tracker only or the same config is also used to download
custom templates when you run "nuclei -update-templates"

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
